packages/assets-controllers/src/NftController.test.ts

```diff
@@ -4813,4 +4831,27 @@ describe('NftController', () => {
 
     expect(updateNftMetadataSpy).not.toHaveBeenCalled();
   });
+
+  describe('getNFTContractInfo', () => {
+    it('fetches NFT collections metadata successfully', async () => {
+      nock(NFT_API_BASE_URL)
+        .get(
+          `/collections?chainId=0x1&contract=${ERC_721_1_ADDRESS}&contract=${ERC_721_2_ADDRESS}`,
+        )
+        .reply(200, {
+          collections: [ERC_721_1_COLLECTION_MOCK, ERC_721_2_COLLECTION_MOCK],
+        });
+
+      const { nftController } = setupController();
+
+      const response = await nftController.getNFTContractInfo(
+        [ERC_721_1_ADDRESS, ERC_721_2_ADDRESS],
+        ChainId.mainnet,
+      );
+
+      expect(response).toStrictEqual({
+        collections: [ERC_721_1_COLLECTION_MOCK, ERC_721_2_COLLECTION_MOCK],
+      });
```

Comment:
What are your thoughts on including the data that this test uses in the test itself? I feel like this would make it easier for a contributor to  read the test from start to finish and understand how this method is designed to work just from reading the tests:

``` suggestion
      const contractAddresses = [
        '0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB',
        '0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB',
      ];
      const collections = [
        {
          id: contractAddresses[0],
          name: 'CryptoPunks',
          slug: 'cryptopunks',
          symbol: 'PUNK',
          imageUrl: 'url',
        },
        {
          id: contractAddresses[1],
          name: 'Kudos',
          slug: 'kudos',
          symbol: 'KUDOS',
          imageUrl: 'url',
        },
      ];
      nock(NFT_API_BASE_URL)
        .get(
          `/collections?chainId=0x1&contract=${contractAddresses[0]}&contract=${contractAddresses[1]}`,
        )
        .reply(200, {
          collections,
        });

      const { nftController } = setupController();

      const response = await nftController.getNFTContractInfo(
        contractAddresses,
        ChainId.mainnet,
      );

      expect(response).toStrictEqual({
        collections,
      });
```

Reply:
Done! Thanks for the suggestion